src/index.js

```diff
@@ -0,0 +1,90 @@
+import Schedule from './module/classSchedule.js';
+import Storage from './module/classStorage.js';
+import Task from './module/classTask.js';
+import './style.css';
+import imgplus from './images/plus.svg';
+import imgreload from './images/reload.svg';
+import imgpencil from './images/pencil.svg';
+
+const myPencil = new Image();
+myPencil.src = imgpencil;
+myPencil.className = 'stop';
+
+const reloadDiv = document.querySelector('.reload');
+const myReload = new Image();
+myReload.src = imgreload;
+reloadDiv.appendChild(myReload);
+
+const plusDiv = document.querySelector('.plus');
+const myPlus = new Image();
+myPlus.src = imgplus;
+plusDiv.appendChild(myPlus);
+
+// event display tasks
+document.addEventListener('DOMContentLoaded', Schedule.displayTasks);
+
+// event add a task
+myPlus.addEventListener('click', () => {
+  const tasktext = document.querySelector('#task-value').value; // get value
+  const code = Storage.getTasks().length + 1;
+  const task = new Task(code, tasktext, false); // create the object
+  Schedule.addTaskToList(task); // add task to the screen list
+  Storage.addTask(task); // Add task to the storage
+  Schedule.clearInput();
+});
+
+// event remove a task
+document.querySelector('.task-list').addEventListener('click', (e) => {
+  if (e.target.classList.contains('stop')) {
+    return;
+  }
+
+  if (e.target.classList.contains('description')) {
+    const targetcode = e.target.nextElementSibling.textContent;
+    const editInput = document.createElement('input');
+    editInput.setAttribute('type', 'text');
+    editInput.style.width = '80%';
+    editInput.style.marginLeft = '5%';
+    editInput.style.fontSize = 'inherit';
+    editInput.value = e.target.innerHTML;
+    editInput.className = 'stop';
+    e.target.innerHTML = '';
+    myPencil.style.marginLeft = '2%';
+    e.target.appendChild(editInput);
+    e.target.appendChild(myPencil);
+    e.target.firstElementChild.select();
+
+    myPencil.addEventListener('click', () => {
+      Storage.editTask(targetcode, editInput.value);
+      Schedule.displayTasks();
+    });
+    return;
+  }
+
+  if (e.target.classList.contains('check')) {
+    const completion = e.target.checked;
+    if (completion) {
+      e.target.nextElementSibling.style = 'text-decoration: line-through';
+    } else { e.target.nextElementSibling.style = 'text-decoration: none'; }
```

Comment:
- [ ] **Required: ** Could you save the state of the checkbox in the localStorage so it can be persistent? When a box is checked, it doesn't reflect on the local storage and the state is lost on reload. Could you fix these?

Reply:
Hello @brytebee , please tell me where I can find that requirement, I see one about Store the updated array of items in local storage but I do not see the requirement about storing the state of the checkbox only with the click on the checkbox or after clicking the reload without clicking on the clear all completed, thanks